packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItem.test.tsx

```diff
@@ -109,6 +104,65 @@ describe('ContextMenuItemChildren', () => {
   });
 });
 
+describe('getItemClassNames', () => {
+  // This test exists to validate that getItemClassNames signature has not changed
```

Comment:
Can you add this instead to contextualMenu.deprecated.tests.ts? (We have this pattern throughout the repo of tests supporting deprecated props/methods)